src/Symfony/Component/Validator/Tests/Constraints/TypeValidatorTest.php

```diff
@@ -166,6 +168,8 @@ public static function getInvalidValues()
             [$file, 'float', 'resource'],
             [$file, 'string', 'resource'],
             [$file, 'object', 'resource'],
+            [['1' => 1], 'list', 'array'],
```

Comment:
This array expression is misleading because it will actually create an array with an integer `1` as key.

```suggestion
            [[1 => 1], 'list', 'array'],
```

In addition to this, I believe we should also add an obvious associative array as invalid value.